streams/src/main/java/org/apache/kafka/streams/KafkaStreaming.java

```diff
@@ -29,6 +29,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.List;
+import java.util.UUID;
```

Comment:
For easy trouble-shooting and debugging, could we add the host name as prefix to the UUID.